doc/how_to/waveform_extractor_to_sorting_analyzer.rst

```diff
@@ -0,0 +1,572 @@
+
+            extractor = load_waveforms(
+                folder="my_waveform_extractor")
+
+    .. grid-item::
+
+        .. code-block:: python
+
+            analyzer = load_sorting_analyzer(
+                folder="my_sorting_analyzer")
+
+
+
+
+Checking basic properties
++++++++++++++++++++++++++
+
+The object contains both a ``sorting`` and a ``recording`` object. These
+can be isolated
+
+
+.. grid:: 2
+
+    .. grid-item::
+
+        .. code-block:: python
+
+            the_recording = extractor.recording
+            the_sorting = extractor.sorting
+
+    .. grid-item::
+
+        .. code-block:: python
+
+            the_recording = analyzer.recording
+            the_sorting = analyzer.sorting
+
+
+
+
+You can then check any ``recording`` or ``sorting`` properties from these objects.
+
+There is much information about the recording and sorting contained in the parent object. E.g. you can get
+the channel locations as follows
+
+.. grid:: 2
+
+    .. grid-item::
+
+        .. code-block:: python
+
+            channel_locations =
+                extractor.get_channel_locations()
+
+    .. grid-item::
+
+        .. code-block:: python
+
+            channel_locations =
+                analyzer.get_channel_locations()
+
+
+
+
+Many properties can be accessed in a similar way
+
+.. grid:: 2
+
+    .. grid-item::
+
+        .. code-block:: python
+
+            extractor.get_num_channels()
+            extractor.get_num_samples()
+            extractor.get_num_segments()
+            extractor.get_probe()
+            extractor.get_probegroup()
+            extractor.get_total_duration()
+            extractor.get_total_samples()
+
+    .. grid-item::
+
+        .. code-block:: python
+
+            analyzer.get_num_channels()
+            analyzer.get_num_samples()
+            analyzer.get_num_segments()
+            analyzer.get_probe()
+            analyzer.get_probegroup()
+            analyzer.get_total_duration()
+            analyzer.get_total_samples()
+
+...while some are simply properties of the object
+
+.. grid:: 2
+
+    .. grid-item::
+
+        .. code-block:: python
+
+            extractor.channel_ids
+            extractor.unit_ids
+            extractor.sampling_frequency
+
+    .. grid-item::
+
+        .. code-block:: python
+
+            analyzer.channel_ids
+            analyzer.unit_ids
+            analyzer.sampling_frequency
+
+
+
+
+You can also find some fundamental properties of the object,
+though these are mostly used internally:
+
+.. grid:: 2
+
+    .. grid-item::
+
+        .. code-block:: python
+
+            extractor.folder
+            extractor.format
+            extractor.is_read_only()
+            extractor.dtype
+            extractor.is_sparse()
+
+    .. grid-item::
+
+        .. code-block:: python
+
+            analyzer.folder
+            analyzer.format
+            analyzer.is_read_only()
+            analyzer.get_dtype()
+            analyzer.is_sparse()
+
+
```

Comment:
Are we missing on the section where the waveform_extractor required the recording or needed to be put into recordingless mode whereas the sorting_analyzer automatically deals with this?